platform_versioned_docs/version-24.1/launch/advanced.mdx

```diff
@@ -43,7 +90,13 @@ reports:
 Run custom code either before or after the execution of the Nextflow script. These fields allow you to enter shell commands.
 
 Pre-run scripts are executed in the nf-launch script prior to invoking Nextflow processes. Pre-run scripts are useful for:
-- Specifying an alternate Nextflow version to use for the run. For example, use `export NXF_VER=24.04.2` to take advantage of [new Nextflow features](https://nextflow.io/blog/2024/nextflow-2404-highlights.html) before a Platform version is released with 24.04.2 as its baseline. 
+- Specifying an alternate Nextflow version to use for the run:
+
+  ```bash 
+  nextflow self-update
```

Comment:
this is only need when updating a version prior to `24.10.0` to version `24.10.0` (or later)

Reply:
@pditommaso you think it's ok to keep in for now, then in six months once everyone has the newer nf-launcher with Nextflow >= 24.10.0 we can remove again?

@llewellyn-sl maybe we can add a comment in the docs to this effect.

I don't think that having this here when it is not needed will have any negative effect.

Reply:
I agree, it's fine